polling/periodic_sequence/activities.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+
+from temporalio import activity
+
+
+@dataclass
+class ComposeGreetingInput:
+    greeting: str
+    name: str
+
+
+@activity.defn
+async def compose_greeting(input: ComposeGreetingInput) -> str:
+    print(f"Invoking activity, attempt number {activity.info().attempt}")
+    # Fail the first 4 attempts, succeed the 5th
+    if activity.info().attempt < 5:
```

Comment:
This will never have more than one attempt, which means this never succeeds (which maybe that's fine, but it makes this `if` unnecessary)

Reply:
Unresolving. You didn't fix this.

Reply:
Fixed